jenkins/prepare-packaged-macos-tests.sh

```diff
@@ -66,7 +66,7 @@ cd mac-test-package
 COUNTER=0
 while [[ $COUNTER -lt 5 ]]; do
 	EC=0
```

Comment:
style issue: when I have to read a shell script, it means that my day has already gone worse than I anticipated. I prefer shell variables to be as self-describing as possible. It took a couple google searches to find that the intent of `EC`. How about `EXIT_CODE` or `ERROR_CODE`?
As a side note, the `|| EC=$?` is clever.

Reply:
I agree about variable names, I'll have that in mind for the future, since it would be out of scope for this PR.